### PR TITLE
Removed any comment that had any form of "To be supplied"

### LINKED ISSUE #50: Remove unwanted comments from System.Windows.Forms code
Feedback from @danmosemsft:

Remove comments like: 
/// <devdoc>
        ///    <para>[To be supplied.]</para>
/// </devdoc>

from ResXResourceWriter.cs and other files.

### PR DESCRIPTION
**Pull request by carlowahlstedt**
Fixes #50.

Simply removed all of the comments that had any form of "To be supplied" in them.

### CODE REVIEW

File: src/System.Windows.Forms/src/System/Resources/ResXFileRef.cs [hunk 1]
```diff
@@ -88,29 +88,20 @@ public class ResXFileRef {
 
 
         /// <include file='doc\ResXFileRef.uex' path='docs/doc[@for="ResXFileRef.FileName"]/*' />
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Resources/ResXFileRef.cs]: the `///<include ` comment is not used either. This could be another issue

**Comment by Tanya-Solyanik** [comment]
@carlowahlstedt Thank you for picking up an issue! Could you please share regular expression that you used to generate these changes. Change of this size is hard to review.
Also I noticed that you replaced an apostrophe in possessive nouns with a different character, it might be better to re-phrase such comments with  no apostrophe.

File: src/System.Windows.Forms/src/System/Resources/ResXFileRef.cs (hunk 1 above)
Comment (carlowahlstedt) [reply src/System.Windows.Forms/src/System/Resources/ResXFileRef.cs]: I almost considered removing this, but figured it was out of scope.

**Comment by carlowahlstedt** [comment]
Didn't use a RegEx, was trying not to [have 100 problems 😆](https://www.xkcd.com/1171/). Honestly I'm not that good with them. I used VS Code to do a multiline search and then replaced in bulk.

Not sure what happened with that apostrophe. I'm suspecting an encoding issue. I made the change on a Mac. When I removed it just now, it's still showing something different in the editor. I'll have to dig deeper.

**Comment by carlowahlstedt** [comment]
Cloned the original repo, looked at the file and it translated that way by default. Please, let me know what you'd like to do in that scenario.

Is there anything else I can do to help with the size of the changes? I was just trying to do everything from that issue at once.

File: src/System.Windows.Forms/src/System/Windows/Forms/WebBrowser.cs [hunk 2]
```diff
@@ -1555,7 +1547,7 @@ private string ReadyNavigateToUrl(Uri url)
                     throw new ArgumentException(string.Format(SR.WebBrowserNavigateAbsoluteUri, "uri"));
                 }
 
-                // Characters outside of US-ASCII may appear in Windows file paths and accordingly they�re allowed in file URIs.
+                // Characters outside of US-ASCII may appear in Windows file paths and accordingly they�re allowed in file URIs.
```
Comment (JuditRose) [inline src/System.Windows.Forms/src/System/Windows/Forms/WebBrowser.cs]: @carlowahlstedt Thanks for the fixes. I went through all the changes manually. Besides the 3 apostrophe issues it LGTM. If it is not trivial to figure out the character encoding I think it's okay to swap these occurrences to the long form for now: isn't -> is not, they're -> they are.

File: src/System.Windows.Forms/src/System/Windows/Forms/AccessibleObject.cs [hunk 3]
```diff
@@ -2611,7 +2608,7 @@ public class AccessibleObject : StandardOleMarshalObject,
 
             int result = NativeMethods.S_FALSE;
             try {
-                // The activityId can be any string. It cannot be null. It isn�t used currently.
+                // The activityId can be any string. It cannot be null. It isnt used currently.
                 result = UnsafeNativeMethods.UiaRaiseNotificationEvent(
```
Comment (Tanya-Solyanik) [inline src/System.Windows.Forms/src/System/Windows/Forms/AccessibleObject.cs]: Consider `is not`

File: src/System.Windows.Forms/src/System/Resources/ResXFileRef.cs (hunk 1 above)
Comment (Tanya-Solyanik) [reply src/System.Windows.Forms/src/System/Resources/ResXFileRef.cs]: Here is the new issue: https://github.com/dotnet/winforms/issues/167

**Comment by zsd4yr** [comment]
> Didn't use a RegEx, was trying not to [have 100 problems 😆](https://www.xkcd.com/1171/). Honestly I'm not that good with them. I used VS Code to do a multiline search and then replaced in bulk.
> 
> Not sure what happened with that apostrophe. I'm suspecting an encoding issue. I made the change on a Mac. When I removed it just now, it's still showing something different in the editor. I'll have to dig deeper.

I made a change on my PC and had the same encoding issue @carlowahlstedt --it happens to the best of us

File: src/System.Windows.Forms/src/System/Windows/Forms/WebBrowser.cs (hunk 2 above)
Comment (carlowahlstedt) [reply src/System.Windows.Forms/src/System/Windows/Forms/WebBrowser.cs]: Fixed this one pointed out here.

File: src/System.Windows.Forms/src/System/Windows/Forms/AccessibleObject.cs (hunk 3 above)
Comment (carlowahlstedt) [reply src/System.Windows.Forms/src/System/Windows/Forms/AccessibleObject.cs]: Fixed this one pointed out here.

**Comment by Tanya-Solyanik** [comment]
> I fixed the two specific instances with the apostrophe mentioned above but there are other instances. I didn't yet dig into why it's doing what it's doing, but might be worth opening an issue to look into it?

Makes sense, please open a new issue for a thorough clean up. Thank you!

**Comment by Tanya-Solyanik** [comment]
@zsd4yr: 

> Would is not be better to actually supply these?

Sure, we can do it as we work with individual files more closely, for example when making changes in a public function, we can also improve the comments.

**Pull request merged**